docs/docs/writing-plugins/macros.mdx

```diff
@@ -107,3 +107,39 @@ custom_python_sources(
     has_type_hints=False,
 )
 ```
+
+## Documenting your macros
+
+Using doc-strings to document your macros, Pants will pick up and present these for the online help text. Also global constants can be documented using `Annotated`.
+
+```python title="pants-plugins/macros.py"
+OUR_GLOBAL_CONSTANT: Annotated[
+  int,
+  """This is our magic number.
```

Comment:
How does indentation work? Do we softwrap?

Reply:
Ah, that would be worth looking into. As of now, we just take it verbatim.